sklearn/mixture/_base.py

```diff
@@ -252,28 +256,35 @@ def fit_predict(self, X, y=None):
 
             lower_bound = -np.inf if do_init else self.lower_bound_
 
-            for n_iter in range(1, self.max_iter + 1):
-                prev_lower_bound = lower_bound
+            if self.max_iter == 0:
+                best_params = self._get_parameters()
+                best_n_iter = 1
```

Comment:
shouldn't it be `best_n_iter = 0` ?

Reply:
You're right, thanks for pointing this out.